pandas/core/common.py

```diff
@@ -2672,6 +2672,9 @@ def is_list_like(arg):
      return (hasattr(arg, '__iter__') and
             not isinstance(arg, compat.string_and_binary_types))
 
+def is_named_tuple(arg):
+    return isinstance(arg, tuple) and hasattr(arg, '_fields')
```

Comment:
`namedtuple` is not a type, so that wouldn't work, unfortunately

FYI: it's an factory function which builds and `eval`s string of a class inherited from `tuple`. Example code here: https://docs.python.org/2/library/collections.html#collections.namedtuple